RI.md

```diff
@@ -0,0 +1,842 @@
+NEWS-1.9.1
+NEWS-1.9.2
+NEWS-1.9.3
+```
+
+```sh
+$ ri ruby: | tail
+syntax/control_expressions.rdoc
+syntax/exceptions.rdoc
+syntax/literals.rdoc
+syntax/methods.rdoc
+syntax/miscellaneous.rdoc
+syntax/modules_and_classes.rdoc
+syntax/pattern_matching.rdoc
+syntax/precedence.rdoc
+syntax/refinements.rdoc
+win32/README.win32
+```
+
+The list of gem pages is available via _name_ `'_gem_name_'`:
+
+```sh
+$ ri nokogiri: | head
+= Pages in gem nokogiri-1.16.2-x86_64-linux
+
+README.md
+lib/nokogiri/css/tokenizer.rex
+```
+
+See also:
+
+- [Option \\--list][8]:
+  lists classes and modules.
+- [Option \\--list-doc-dirs][12]:
+  lists `ri` source directories.
+
+## `ri` Information
+
+With certain options,
+an `ri` command may display information other than documents or lists:
+
+- [Option \\--help or -h][13]:
+  Shows `ri` help text.
+- [option \\--version or -v][14]:
+  Shows `ri` version.
+- [Option \\--dump=FILEPATH][15]:
+  Shows dump of `ri` cache file at the given filepath.
+
+## Static Mode
+
+In static mode, `ri` shows a response and exits.
+
+In general, `ri` responds in static mode
+if the command gives a _name_:
+
+```sh
+$ ri Array | head
+= Array < Object
+
+------------------------------------------------------------------------
+= Includes:
+Enumerable (from ruby core)
+
+(from ruby core)
+------------------------------------------------------------------------
+An Array is an ordered, integer-indexed collection of objects, called
+elements.  Any object may be an Array element.
+```
+
+`ri` also responds in static mode when certain options are given,
+even when no _name_ is given;
+see [ri Information][16].
+
+## Interactive Mode
+
+In general, `ri` responds to a command in interactive mode
+if the command has no arguments:
+
+```sh
+$ ri
+Enter the method name you want to look up.
+You can use tab to autocomplete.
+Enter a blank line to exit.
+>>
+
+```
+
+A command in interactive mode are similar to one in static mode,
+except that it:
+
+- Omits command word `ri`; you just type the _name_.
+- Omits options; in interactive mode the only options in effect
+  are those taken from environment variable `RI`.
+  See [Options][17].
+- Supports tab auto-completion for the name of a class, module, or method;
+  when, for example, you type `"Arr\t"` (here `"\t` represents the tab character),
+  `ri` "completes" the text as `'Array '`.
+
+See also [ri at the Ready][18].
+
+## Pager
+
+Because `ri` output is often large,
+`ri` by default pipes it to a _pager_,
+which is the program whose name is the first-found among:
+
+- The value of `ENV['RI_PAGER']`.
+- The value of `ENV['PAGER']`.
+- `'pager'`.
+- `'less'`.
+- `'more'`.
+
+If none is found, the output goes directly to `$stdout`, with no pager.
+
+If you set environment variable `RI_PAGER` or `PAGER`,
+its value should be the name of an executable program
+that will accept the `ri` output (such as `'pager'`, `'less'`, or `'more'`).
+
+See also [Output Filters][19].
+
+## \Options
```

Comment:
This title and many `Options` in subtitles under this section all have backslashes, which don't seem to be necessary. Can we remove them?

Reply:
Fixed.